harvard-university-of-worcester.csl

```diff
@@ -0,0 +1,366 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="sort-only" version="1.0" class="in-text" et-al-use-first="1" et-al-min="3" default-locale="en-GB" page-range-format="expanded">
+  <info>
+    <title>Harvard - University of Worcester</title>
+    <id>http://www.zotero.org/styles/harvard-university-of-worcester</id>
+    <link rel="self" href="http://www.zotero.org/styles/harvard-university-of-worcester"/>
+    <link rel="documentation" href="http://www.worc.ac.uk/ils/documents/Harvard_referencing"/>
+    <author>
+      <name>Christopher Hill</name>
+      <uri>http://www.mendeley.com/profiles/christopher-hill7/</uri>
+    </author>
+    <contributor>
+      <name>Sarah Pittaway</name>
+    </contributor>
+    <category citation-format="author-date"/>
+    <summary>Harvard author-date style edited for the University of Worcester</summary>
+    <updated>2015-08-08T21:12:31+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names delimiter=", " variable="editor">
+      <name and="symbol" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" suffix=")" prefix=" ("/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text form="short" text-case="capitalize-first" term="anonymous" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter-precedes-last="never"/>
+      <et-al font-style="normal"/>
+      <substitute>
+        <names variable="translator"/>
```

Comment:
you left ```<names variable="translator"/>``` here in the author-short macro but don't have it in author (and I'd guess you don't want it). Is that an oversight? Otherwise this looks ready to merge.